task/06-conditions-n-loops-tasks.js

```diff
@@ -396,10 +458,22 @@ function getCommonDirectoryPath(pathes) {
  *
  */
 function getMatrixProduct(m1, m2) {
-    throw new Error('Not implemented');
+  let res = Array(m1.length);
```

Comment:
Use `new Array` instead of `Array`. It is a potential bug. Why? If you see MDN, you will not find anything about `Array` function.